### PR TITLE
e2e tests

### PR DESCRIPTION
**Pull request by SimonasB88**
@tomas862 
- What do you think, what frequency to set for ~1h tests run? (https://github.com/mollie/PrestaShop/blob/e2e-tests/.github/workflows/midnight_status_check.yml)
- I have to add 2 variable secrets in this repo. How can I?

### CODE REVIEW

File: cypress/integration/ps177/Mollie_ps1770_demo_bancontact.js [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+//       cy.get('.module-import-start').attachFile('fixture.zip', { subjectType: 'drag-n-drop' }).wait(15000)
+//       cy.get('.module-import-success-msg').should('be.visible').as('Module installed!')
+//
+// })
+    it('Enabling Bancontact payment [Orders API]', () => {
+      cy.mollie_test17_admin()
+      cy.login_mollie17_test()
+      cy.get('#subtab-AdminMollieModule > .link').click()
+      cy.get('#MOLLIE_API_KEY_TEST').clear().type('test_pACCABA9KvWGjvW9StKn7QTDNgMvzh',{delay: 0})
+      cy.get('#MOLLIE_PROFILE_ID').clear().type('pfl_jUAQPFDdTR',{delay: 0})
```
Comment (tomas862) [inline cypress/integration/ps177/Mollie_ps1770_demo_bancontact.js:35]: same here

File: cypress/integration/ps177/Mollie_ps1770_demo_bancontact.js [hunk 2]
```diff
@@ -0,0 +1,163 @@
...
+      cy.get('#subtab-AdminEmails > .link').click()
+      cy.get('#email_logs_grid_table > tbody > :nth-child(2) > :nth-child(4)').should('include.text','order_conf')
+      cy.get('#email_logs_grid_table > tbody > :nth-child(1) > :nth-child(4)').should('include.text','payment')
+})
+    it('Enabling Bancontact payment [Payments API]', () => {
+      cy.mollie_test17_admin()
+      cy.login_mollie17_test()
+      cy.get('#subtab-AdminMollieModule > .link').click()
+      cy.get('#MOLLIE_API_KEY_TEST').clear().type('test_pACCABA9KvWGjvW9StKn7QTDNgMvzh',{delay: 0})
+      cy.get('#MOLLIE_PROFILE_ID').clear().type('pfl_jUAQPFDdTR',{delay: 0})
```
Comment (tomas862) [inline cypress/integration/ps177/Mollie_ps1770_demo_bancontact.js:104]: same

File: cypress/integration/ps177/Mollie_ps1770_demo_credictard_payment_api_order_api.js [hunk 3]
```diff
@@ -0,0 +1,268 @@
...
+//         response: 500
+// })
+//
+//       cy.get('#subtab-AdminParentModulesSf > :nth-child(1)').click()
+//       cy.get('#subtab-AdminModulesSf > .link').click()
+//       cy.get('#page-header-desc-configuration-add_module').click()
+//       cy.get('.module-import-start').attachFile('fixture.zip', { subjectType: 'drag-n-drop' }).wait(15000)
+//       cy.get('.module-import-success-msg').should('be.visible').as('Module installed!')
+//
+// })
```
Comment (tomas862) [inline cypress/integration/ps177/Mollie_ps1770_demo_credictard_payment_api_order_api.js:29]: remove commented code from everywhere

File: cypress/integration/ps177/Mollie_ps1770_demo_credictard_payment_api_order_api.js [hunk 4]
```diff
@@ -0,0 +1,268 @@
...
+    // Checking the Credit Card enabling
+    it('Checking the Credit Card Payment API method successfully enabling BO', () => {
+      cy.mollie_test17_admin()
+      cy.login_mollie17_test()
+      cy.get('#subtab-AdminMollieModule > .link').click()
+      cy.get('#MOLLIE_API_KEY_TEST').clear().type('test_pACCABA9KvWGjvW9StKn7QTDNgMvzh',{delay: 0})
+      cy.get('#MOLLIE_PROFILE_ID').clear().type('pfl_jUAQPFDdTR',{delay: 0})
```
Comment (tomas862) [inline cypress/integration/ps177/Mollie_ps1770_demo_credictard_payment_api_order_api.js:36]: same

File: cypress/integration/ps177/Mollie_ps1770_demo_klarna_order_api.js [hunk 5]
```diff
@@ -0,0 +1,166 @@
+/// <reference types="Cypress" />
+context('Klarna [Pay Later, Slice it] payment automation PS1770 Mollie Orders API', () => {
+  beforeEach(() => {
+    cy.viewport(1920,1080)
+  })
+//     it('Uploading the latest module-artifact to the Prestashop', () => {
```
Comment (tomas862) [inline cypress/integration/ps177/Mollie_ps1770_demo_klarna_order_api.js:6]: remove commented code

File: .github/workflows/midnight_status_check.yml [hunk 6]
```diff
@@ -0,0 +1,85 @@
+name: Regular midnight test runs
+on:
+  schedule:
+    #- cron:  '*/5 * * * *'
+      #- cron:  '0 0 * * 3,4,5'
```
Comment (tomas862) [inline .github/workflows/midnight_status_check.yml:5]: why time is commented ?

File: cypress/integration/ps17/Mollie_ps1752_demo_klarna_order_api.js [hunk 7]
```diff
@@ -0,0 +1,212 @@
...
+      cy.mollie_1752_test_login()
+      cy.visit('https://demo.invertus.eu/clients/mollie17-test/admin1/index.php?controller=AdminEmails&token=023927e534d296d1d25aab2eaa409760')
+      cy.get('tbody > :nth-child(2) > :nth-child(4)').should('include.text','order_conf')
+      cy.get('tbody > :nth-child(1) > :nth-child(4)').should('include.text','payment')
+})
+it('Setuping the Order API method in BO [Slice it]', () => {
+      cy.mollie_1752_test_demo_module_dashboard()
+      cy.mollie_1752_test_login()
+      cy.get('#MOLLIE_API_KEY_TEST').clear().type('test_pACCABA9KvWGjvW9StKn7QTDNgMvzh',{delay: 0})
+      cy.get('#MOLLIE_PROFILE_ID').clear().type('pfl_jUAQPFDdTR',{delay: 0})
```
[the remark this answers is not in the document]
Comment (SimonasB88) [reply cypress/integration/ps17/Mollie_ps1752_demo_klarna_order_api.js]: done

File: cypress/integration/ps177/Mollie_ps1770_demo_bancontact.js (hunk 1 above)
Comment (SimonasB88) [reply cypress/integration/ps177/Mollie_ps1770_demo_bancontact.js]: done

File: cypress/integration/ps177/Mollie_ps1770_demo_bancontact.js (hunk 2 above)
Comment (SimonasB88) [reply cypress/integration/ps177/Mollie_ps1770_demo_bancontact.js]: done

File: cypress/integration/ps177/Mollie_ps1770_demo_credictard_payment_api_order_api.js (hunk 3 above)
Comment (SimonasB88) [reply cypress/integration/ps177/Mollie_ps1770_demo_credictard_payment_api_order_api.js]: done

File: cypress/integration/ps177/Mollie_ps1770_demo_credictard_payment_api_order_api.js (hunk 4 above)
Comment (SimonasB88) [reply cypress/integration/ps177/Mollie_ps1770_demo_credictard_payment_api_order_api.js]: done

File: cypress/integration/ps177/Mollie_ps1770_demo_klarna_order_api.js (hunk 5 above)
Comment (SimonasB88) [reply cypress/integration/ps177/Mollie_ps1770_demo_klarna_order_api.js]: done

File: .github/workflows/midnight_status_check.yml (hunk 6 above)
Comment (SimonasB88) [reply .github/workflows/midnight_status_check.yml:5]: haven't decided the frequency of 1h testing

**Pull request merged**